docs/platforms/android/migration/index.mdx

```diff
@@ -92,6 +92,10 @@ You may also use `LifecycleHelper.close(token)`, e.g. in case you need to pass t
 
 - The required Sentry version in version `8.0.0` of the SDK remains unchanged. [Sentry's version >= v22.12.0](https://github.com/getsentry/self-hosted/releases) is required. This only applies to self-hosted Sentry. If you are using [sentry.io](https://sentry.io), no action is needed.
 
+### Sentry Android SDK Compatibility
+
+You must use Sentry Gradle plugin 5.+ together with the Sentry Android SDK 8.+ otherwise, it might crash at runtime due to missing dependencies.
+
```

Comment:
`m` in previous version bumps, we called this out a bit differently, see in the [v7 docs](https://github.com/getsentry/sentry-docs/pull/12878/files#diff-da010a31b74ddc2c3a51e0f534e3372c8686ad5bcffa7b2c7f6d958f93045f03R154) for example. I'm wondering if we should just add a generic section to the top of the page that sticks there